### PR TITLE
Versions Widget - several active versions displayed after restoring c…

### PR DESCRIPTION
**Pull request by ashklianko**
…ontent #4830

-Fixed incorrect setting of active version
-Removed 'revertable' prop from VersionHistoryItem that doesn't belong there
-Updated styling for sorted/permissions items to be not interactable

### CODE REVIEW

File: modules/lib/src/main/resources/assets/js/app/view/context/widget/version/VersionHistoryListItem.ts [hunk 1]
```diff
@@ -33,11 +33,11 @@ export class VersionHistoryListItem
     private createVersionViewer(): VersionHistoryListItemViewer {
         const versionViewer: VersionHistoryListItemViewer = new VersionHistoryListItemViewer();
 
-        if (this.version.isRevertable() || this.version.isActive()) {
+        if (this.isInteractableItem()) {
             this.addOnClickHandler(versionViewer);
         }
 
-        if (this.version.isRevertable()) {
+        if (this.isInteractableItem()) {
```
Comment (alansemenov) [inline modules/lib/src/main/resources/assets/js/app/view/context/widget/version/VersionHistoryListItem.ts]: Any reason for not joining these two `if`s?

Comment (ashklianko) [reply modules/lib/src/main/resources/assets/js/app/view/context/widget/version/VersionHistoryListItem.ts]: Missed it, will fix

**Review by alansemenov — CHANGES_REQUESTED** [review]
It seems to work fine, but there's a new bug. 

After sorting is changed on an item selected in the grid, Version History widget doesn't get refreshed, so you don't see the new version created after the change of sorting. You have to unselect the row, then select it again to see the updates.

**Comment by ashklianko** [comment]
Updated

**Review by alansemenov — APPROVED** [review]

**Pull request merged**